controllers/packagebundle_controller_watch_test.go

```diff
@@ -29,7 +29,7 @@ func TestPackageBundleReconciler_mapBundleReconcileRequests(t *testing.T) {
 			return nil
 		})
 	bm := bundlefake.NewBundleManager()
-	sut := NewPackageBundleReconciler(mockClient, nil, mockBundleClient, bm, logr.Discard())
+	sut := NewPackageBundleReconciler(mockClient, nil, mockBundleClient, bm, nil, logr.Discard())
```

Comment:
Maybe let's try to use optional parameters similar to this: https://github.com/aws/eks-anywhere/blob/8bd0a318a0acd668c14166dfd9cac910653f347e/pkg/curatedpackages/packageclient.go#L29